spec/03-plugins/40-ai-response-transformer/02-integration_spec.lua

```diff
@@ -210,7 +210,7 @@ for _, strategy in helpers.all_strategies() do if strategy ~= "cassandra" then
         server {
             server_name llm;
             listen ]]..MOCK_PORT..[[;
-            
+
```

Comment:
Code Style: We're not adding any **changes** to this file. Perhaps we should leave these indents to minimize the changed scope.

Reply:
This is automatically done based on the [.editorconfig](https://github.com/Kong/kong/blob/master/.editorconfig)

Maybe... we should turn on the linters validation based on that file to ensure they do not randomly show up.